src/openapi/streaming/subscription.js

```diff
@@ -331,7 +346,7 @@ Subscription.prototype.reset = function() {
 			return;
 	}
 
-	this.nextAction = null;
+    this.queue.reset();
```

Comment:
Another indentation problem..
We should convert to spaces and add eslint eventually but let's hold off till this is in..